griptape/tasks/choice_control_flow_task.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import Callable
+from attrs import define, field
+
+from griptape.artifacts import BaseArtifact, ErrorArtifact, TaskArtifact, ListArtifact
+from griptape.tasks import BaseTask
+from griptape.tasks import BaseControlFlowTask
+
+
+@define
+class ChoiceControlFlowTask(BaseControlFlowTask):
```

Comment:
Do we need a dedicated Task for this? What if we added `control_flow_fn` to `BaseTask`?

Reply:
future proof if there is other types of control flow tasks